articles/tokens/refresh-token.md

```diff
@@ -1,152 +1,173 @@
 
-For more information on the types of access tokens used by Auth0, see [Tokens](/tokens).
+Usually, a user will need a new access token only after the previous one expires, or when gaining access to a new resource for the first time.
 
-If you are new to refresh tokens, you can learn more about them in this blog post: [Refresh Tokens: When to Use Them and How They Interact with JWTs](https://auth0.com/blog/refresh-tokens-what-are-they-and-when-to-use-them/).
+Refresh tokens are subject to strict storage requirements to ensure that they are not leaked. Also, [Refresh tokens can be revoked](#revoke-a-refresh-token) by the Authorization Server.
 
-## Introduction
+## Overview
 
-The response of an [authentication request](/protocols) can result in an `id_token` (JWT) being issued by Auth0. This token is used to make authenticated calls to a secured API. JWTs have an expiration date indicated by the `exp` claim (among other security measures, like signing). Applications that are installed locally on a device (such as a desktop or smartphone) may want to avoid asking the user to enter their credentials each time this token expires.
+The response of an [authentication request](/api-auth) can result in an `access_token` and/or an `id_token` being issued by Auth0. The  `access_token` is used to make authenticated calls to a secured API, while the `id_token` contains user profile attributes represented in the form of _claims_. Both JWTs have an expiration date indicated by the `exp` claim (among other security measures, like signing).
 
-A refresh token allows the application to request Auth0 to issue a new `id_token` directly, without needing to re-authenticate the user. This will work as long as the refresh token has not been revoked.
+A refresh token allows the application to request Auth0 to issue a new `access_token` or `id_token` directly, without needing to re-authenticate the user. This will work as long as the refresh token has not been revoked.
 
-Refresh tokens can be issued for each combination of __app__, __user__ and __device__. Once the Auth0 refresh token is issued, the values of the client, user, and device set during its creation cannot be changed.
-
-::: panel-warning Secure Storage
+::: panel-warning Security Warning
 Refresh tokens must be stored securely by an application since they allow a user to remain authenticated essentially forever.
 :::
 
-Refresh tokens can be [obtained](#obtain-a-refresh-token) or [revoked](#revoke-a-refresh-token-using-the-management-api) programmatically through the Auth0 API.
+Refresh tokens can be [obtained](#obtain-a-refresh-token) or [revoked](#revoke-a-refresh-token-with-a-request) programmatically through the Auth0 API.
 
-They can also be viewed and revoked [from the dashboard](#revoke-a-refresh-token-in-the-dashboard).
+They can also be viewed and revoked [from the dashboard](#revoke-a-refresh-token-using-the-dashboard).
 
-## Obtain a Refresh Token
+## Get a Refresh Token
```

Comment:
Here things start to diverge a lot, expected, from the legacy flow. Maybe it's worth to keep the old doc under `/legacy/refresh-tokens` and link from this one.

"This article was rewritten to the new Refresh Token story introduced by the API Authorization Features. If you are looking for the legacy Refresh Token docs you can check here (link)"

Can you sync with Nathan to validate this? We agreed on doing this in the past but never got a doc worth doing it. This can be the first.